src/app/screens/Settings.tsx

```diff
@@ -165,6 +125,48 @@ function Settings() {
           )}
         </Setting>
       </div>
+      <h2 className="mt-12 text-2xl font-bold dark:text-white">
+        Personal data
+      </h2>
+      <div className="mb-6 text-gray-700 dark:text-gray-300 text-sm">
+        Payees can request for additional data to be sent with a payment. This
```

Comment:
reading this again my thought was if "request" is too hard, and if "optional" should be mentioned.
Payees can optionally request additional data to be sent with a payment.

I don't remember what we said so far :)